pkg/commands/git_commands/commit_test.go

```diff
@@ -63,20 +64,23 @@ func TestCommitCommitCmdObj(t *testing.T) {
 		{
 			testName:             "Commit",
 			summary:              "test",
+			verify:               false,
 			configSignoff:        false,
 			configSkipHookPrefix: "",
 			expectedArgs:         []string{"commit", "-m", "test"},
 		},
 		{
 			testName:             "Commit with --no-verify flag",
 			summary:              "WIP: test",
+			verify:               true,
```

Comment:
If we do what I suggest above (keeping the prefix logic in `CommitCmdObj`), then it would be good to add two more test cases here for the different combinations of prefix vs flag.

Reply:
See [97e05e5](https://github.com/jesseduffield/lazygit/pull/4374/commits/97e05e5febcf61b298c60ab156ccf7beb1fd2a6f)